ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -2401,98 +2772,6 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
-    {
-      id: 'P12S The Classical Concepts PS marker',
-      type: 'HeadMarker',
-      netRegex: {},
-      run: (data, matches) => {
-        const id = getHeadmarkerId(data, matches);
-        const psMarkerMap: { [id: string]: PlaystationMarker } = {
-          [headmarkers.playstationCircle]: 'circle',
-          [headmarkers.playstationTriangle]: 'triangle',
-          [headmarkers.playstationSquare]: 'square',
-          [headmarkers.playstationCross]: 'cross',
-        } as const;
-        const marker = psMarkerMap[id];
-        if (marker !== undefined)
-          data.classicalMarker[matches.target] = marker;
-      },
-    },
-    {
-      id: 'P12S The Classical Concepts Alpha Beta',
-      type: 'GainsEffect',
-      netRegex: { effectId: ['DE8', 'DE9'] },
-      run: (data, matches) => {
-        data.classicalAlphaBeta[matches.target] = matches.effectId === 'DE8' ? 'alpha' : 'beta';
-      },
-    },
-    {
-      id: 'P12S The Classical Concepts',
-      type: 'Ability',
-      netRegex: { id: '8331', source: 'Pallas Athena', capture: false },
-      delaySeconds: 2,
-      durationSeconds: (data) => data.classicalCounter === 1 ? 9 : 16,
-      suppressSeconds: 1,
-      infoText: (data, _matches, output) => {
-        const marker = data.classicalMarker[data.me];
-        const tether = data.classicalAlphaBeta[data.me];
-        if (marker === undefined || tether === undefined)
-          return;
-        return output.text!({ marker: output[marker]!(), shape: output[tether]!() });
-      },
-      outputStrings: {
-        text: {
-          en: '${marker} (block ${shape})',
-          ja: '${marker} + ${shape}', // FIXME
-        },
-        circle: {
-          en: 'Circle',
-          de: 'Kreis',
-          fr: 'Cercle',
-          ja: 'まる',
-          cn: '圆圈',
-          ko: '동그라미',
-        },
-        triangle: {
-          en: 'Triangle',
-          de: 'Dreieck',
-          fr: 'Triangle',
-          ja: 'さんかく',
-          cn: '三角',
-          ko: '삼각',
-        },
-        square: {
-          en: 'Square',
-          de: 'Viereck',
-          fr: 'Carré',
-          ja: 'しかく',
-          cn: '方块',
-          ko: '사각',
-        },
-        cross: {
-          en: 'X',
-          de: 'X',
-          fr: 'Croix',
-          ja: 'バツ',
-          cn: 'X',
-          ko: 'X',
-        },
-        alpha: {
-          // Deliberately say "pyramid" here to differentiate from "triangle" marker.
-          en: 'pyramid',
-        },
-        beta: {
-          en: 'cube',
-        },
-      },
-    },
-    {
-      id: 'P12S The Classical Concepts Move',
```

Comment:
> I think you shouldn't remove this one. (And if anything, we could use a bait proteans call for this.)

Yup yup.  Readding it.  I'll also add a bait proteans call.  I think the right thing to trigger on for that is the actual tethers between 8 red/yellow concepts and the player(s) themselves, since that's the point at which its safe to move into bait position.  I think I'm also going to keep conceptDebuff alive until that trigger and create two separate outputs (e.g. baitAlphaDebuff and baitBetaDebuff) - although I'll initially set them both to 'Bait Protean' by default and let folks customize.  I think the main Aether PF strat right now is to have alphas bait north(ish) and betas bait south(ish) so that'll give some flexibility.  

https://cdn.discordapp.com/attachments/1110685852545339543/1122757295235940393/image.png

Reply:
Oh that's interesting.  My group has been just doing "if you're the more north tether, you bait north (and I guess call if you're horizontal)" but alpha vs beta beta location makes a lot of sense as a way to coordinate.

The bait timing is different on the two classicals too.  On classical 2, I definitely agree that once the purple tethers appear on the players, then it's time to bait.  On classical 1, it's maybe more when the green tethers disappear (or sometime around there, maybe you have a better idea).

(Now that I look at it, I also wonder if there's a better trigger for the "move" call than using a delay? Maybe it makes more sense to use the actual initial protean ability + suppress?)

Reply:
> The bait timing is different on the two classicals too. On classical 2, I definitely agree that once the purple tethers appear on the players, then it's time to bait. On classical 1, it's maybe more when the green tethers disappear (or sometime around there, maybe you have a better idea).
> 
> (Now that I look at it, I also wonder if there's a better trigger for the "move" call than using a delay? Maybe it makes more sense to use the actual initial protean ability + suppress?)

Yeah, I've been rewatching vod, and I do think the regex should be different for classical1 & classical2.
For classical1, I think when the playstation tethers disappear, it should probably be 'Avoid Shapes -> Bait Protean'.  The move call is more tricky.  The proteans are baited right at the end of the Palladian Ray cast, and the damage starts to be applied almost immediately, so if you're not already moving, you get ticked.  (It's even worse than Twisters).  Calling 'move' at the end of the cast is probably too late and would require instant reaction time, so I think I'd probably still lean toward doing a delay from the 20 line, probably something like cast time minus 1 sec.

And yeah, for classical2, using the tethers to bait proteans is right.  I think the Avoid Shapes call should probably be the same timing as the 'Move' call for classical1.